packages/components/psammead-navigation/src/index.jsx

```diff
@@ -0,0 +1,231 @@
+
+const StyledNav = styled.nav`
+  background-color: ${C_POSTBOX};
+  position: relative;
+`;
+
+const SkipLink = styled.a`
+  position: absolute;
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  background-color: #ffffff;
+  border: ${SKIP_LINK_BORDER} solid #000;
+  color: ${SKIP_LINK_COLOR};
+  text-decoration: none;
+  font-family: ${GEL_FF_REITH_SANS};
+  ${props => (props.script ? getPica(props.script) : '')};
+
+  &:focus {
+    clip-path: none;
+    clip: auto;
+    top: ${SKIP_LINK_TOP_POSITION_LARGE};
+    left: ${GEL_SPACING_DBL};
+    height: auto;
+    width: auto;
+
+    @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+      top: ${SKIP_LINK_TOP_POSITION_SMALL};
+      left: ${GEL_SPACING};
+    }
+  }
+`;
+
+const StyledUnorderedList = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  position: relative;
+  overflow: hidden;
+`;
+
+const StyledListItem = styled.li`
+  display: inline-block;
+  position: relative;
+  z-index: 2;
+
+  /* Trick to display a border between the list items when it breaks into multiple lines, which takes the full width */
+  &::after {
+    content: ' ';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+    border-bottom: 1px solid ${BORDER_COLOR};
+    z-index: -1;
+
+    ${({ dir }) =>
+      dir === 'ltr'
+        ? css`
+            left: 0;
+          `
+        : css`
+            right: 0;
+          `};
+  }
+`;
+
+const ListItemBorder = css`
+  content: '';
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-bottom: ${GEL_SPACING_HLF} solid ${C_WHITE};
+`;
+
+const StyledLink = styled.a`
+  ${props => (props.script ? getPica(props.script) : '')};
+  font-family: ${GEL_FF_REITH_SANS};
+  color: ${C_GHOST};
+  cursor: pointer;
+  text-decoration: none;
+  display: inline-block;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING_DBL};
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  }
+
+  &:hover::after {
+    ${ListItemBorder}
+  }
+
+  &:focus::after {
+    ${ListItemBorder}
+    top: 0;
+    border: 0.25rem solid ${C_WHITE};
+  }
+`;
+
+const StyledSpan = styled.span`
+  ${props => (props.script ? getPica(props.script) : '')};
+  font-family: ${GEL_FF_REITH_SANS};
+  color: ${C_GHOST};
+  display: inline-block;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING_DBL};
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  }
+
+  &::before {
+    ${ListItemBorder}
+  }
+`;
+
+const CurrentItem = ({ children: link, script, currentPageText }) => (
+  <Fragment>
+    <StyledSpan
+      // eslint-disable-next-line jsx-a11y/aria-role
+      role="text"
+      script={script}
+    >
+      <VisuallyHiddenText>{currentPageText},</VisuallyHiddenText>
```

Comment:
```suggestion
      <VisuallyHiddenText>{currentPageText}, </VisuallyHiddenText>
```
Would this be important for the screen reader UX? I'm not sure

Reply:
Yes, there should be a space.